src/platform/testing/e2e/nightwatch-commands/evaluateRadioButtons.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable func-names, prefer-arrow-callback */
+
+/**
+ * Evaluate radio buttons for basic forward keyboard functionality:
+ *
+ * Radio groups can receive keyboard focus by pressing TAB
+ * Each radio button can be checked by pressing the DOWN or
+ * RIGHT arrow keys.
+ *
+ * ```javascript
+ *  this.demoTest = function (client) {
+ *    client.evaluateRadioButtons([
+ *      "input[type="radio"]#1",
+ *      "input[type="radio"]#2",
+ *      "input[type="radio"]#3"
+ *    ], client.Keys.ARROW_DOWN);
+ *  };
+ * ```
+ *
+ * @method evaluateRadioButtons
+ * @param {string} selectorArray The array of radio buttons to be evaluated
+ * @param {object} arrowPressed Nightwatch Keys object. Expects ARROW_DOWN || ARROW_RIGHT.
+ * @api commands
+ */
+exports.command = function evaluateRadioButtons(selectorArray, arrowPressed) {
```

Comment:
What do you think about `allyEvaluateRadioButtons` to make it clearer what it's evaluating?